chain/epoch-manager/src/reward_calculator.rs

```diff
@@ -353,6 +419,98 @@ mod tests {
         }
     }
 
+    // Test rewards when some validators are only responsible for endorsements
+    #[test]
+    fn test_reward_stateless_validation() {
```

Comment:
I think validators responsible for all 3 roles are also worth unit testing, because that's what we will see on MVP - top validators will produce blocks and chunks and validate all the shards.

Reply:
`test1` has all three roles in `test_reward_stateless_validation`